app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
 
+    def __init__(self, name, age):
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
+
+
+def create_person_list(people: list):
+    list_of_person = [Person(pers["name"], pers["age"]) for pers in people]
```

Comment:
Please, do not shorten variable names
`pers` -> `person`
`list_of_person` -> `list_of_people`